client/src/components/moecules/ProductItem.tsx

```diff
@@ -0,0 +1,137 @@
+import styled from '@emotion/styled';
+import { TProduct } from '@fleamarket/common';
+import { IconButton } from '@mui/material';
+import Dir from './ImageLarge.png';
+import FavoriteBorderIcon from '@mui/icons-material/FavoriteBorder';
+import FavoriteIcon from '@mui/icons-material/Favorite';
+import ChatBubbleOutlineIcon from '@mui/icons-material/ChatBubbleOutline';
+import {
+  TEXT_LINK_MEDIUM,
+  TEXT_SMALL,
+  TEXT_LINK_SMALL,
+  COLOR,
+} from '@constants/style';
+import { css } from '@emotion/css';
+
+interface IProps {
+  product: TProduct;
+}
+
+const ProductItem: React.FC<IProps> = ({ product }) => {
+  const { id, title, price, isLike, likes, chats, locationName } = product;
+
+  return (
+    <>
+      <ContainerDiv>
+        <div>
+          <Image src={Dir}></Image>
+        </div>
+        <ContentDiv>
+          <TitleHeader>
+            <TitleSpan
+              className={css`
+                ${TEXT_LINK_MEDIUM}
+              `}
+            >
+              {title}
+            </TitleSpan>
+            <IconButton>
+              {isLike ? (
+                <FavoriteIcon sx={{ color: COLOR.error }} />
+              ) : (
+                <FavoriteBorderIcon />
+              )}
+            </IconButton>
+          </TitleHeader>
+
+          <div>
+            <span
+              className={css`
+                ${TEXT_SMALL}
+              `}
+            >
+              {locationName} {`∙ 2분전`}
+            </span>
+          </div>
+
+          <div style={{ paddingTop: '8px' }}>
+            <span
+              className={css`
+                ${TEXT_LINK_SMALL}
+              `}
+            >
+              {price === 0 ? '무료나눔' : price?.toLocaleString()}원
+            </span>
+          </div>
+
+          <IconDiv>
+            {chats > 0 ? (
+              <ChatWrapper
+                className={css`
+                  ${TEXT_SMALL}
+                `}
+              >
+                <ChatBubbleOutlineIcon />
+                <span>{chats}</span>
+              </ChatWrapper>
+            ) : null}
```

Comment:
```suggestion
            {chats > 0 && (
              <ChatWrapper
                className={css`
                  ${TEXT_SMALL}
                `}
              >
                <ChatBubbleOutlineIcon />
                <span>{chats}</span>
              </ChatWrapper>
            )}
```
와 같은 방법으로 사용하면 삼항연산자 대신 더 좋게 표현할 수 있을 것 같습니다!

Reply:
api 명세 type에서 해당 필드가 undefinde도 될 수 있다고 처음에 정의해서 위와 같은 방식으로 했는데 디폴트값을 주기로 했으니 위의 방식이 더 깔끔하네요 ㅎㅎ 감사합니다 수정했습니다~!